packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -660,6 +660,27 @@ export default class TabDisks extends Component {
     }
   }
 
+  _areVdisSrsIllegal = createSelector(
```

Comment:
Hardly maintainable when you don't remember what "illegal" means. What about `_areSrsOnSameHost` and reverse the conditions to get rid of negations? (open to discussion, as always)

Reply:
`_areSrsNotOnSameHost` ? to avoid negative conditions in render